src/miner.cpp

```diff
@@ -30,10 +30,16 @@
 #include "validation/validation.h"
 #include "validationinterface.h"
 
+#include <algorithm>
+#include <boost/thread.hpp>
 #include <boost/tuple/tuple.hpp>
 #include <queue>
 #include <thread>
 
+// Track timing information for Score and Pacakge mining.
```

Comment:
spelling: Package

Also, I'm just going from top to bottom here so may be missing something but shouldn't these be members of BlockAssembler?

Reply:
fixed spelling

as for being part of block assembler, i don't think it makes
sense...these are global vars used to track peformance in the
miner_tests.cpp peformancetest section.